test/GraphStoreProtocolTest.cpp

```diff
@@ -18,57 +18,174 @@ using namespace ad_utility::url_parser::sparqlOperation;
 using Var = Variable;
 using TC = TripleComponent;
 
+namespace {
+auto ClearGraph = [](SparqlTripleSimpleWithGraph::Graph graph)
```

Comment:
Commen/document: 
"A matcher that matches a ParsedQuery that is an updated that deletes all triples from the given `graph`"